packages/protobuf-test/src/gen/js/extra/name-clash_pb.js

```diff
@@ -17,6 +17,19 @@
 /* eslint-disable */
 
 import { proto3 } from "@bufbuild/protobuf";
+import { User as User$1 } from "./example_pb.js";
+
+/**
+ * This message class will clash with the imported message class
+ *
+ * @generated from message spec.User
+ */
+export const User = proto3.makeMessageType(
```

Comment:
With the fix https://github.com/bufbuild/protobuf-es/pull/688/commits/729b9f3ac4eb79c74f934f3cda462c27fbbcac35, the import is aliased as expected.